subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -1522,10 +1522,13 @@ static void conn_cleanup(struct ll_conn *conn)
 		ll_rx_put(rx->hdr.link, rx);
 	}
 
+#if defined(CONFIG_BT_CTLR_LOW_LAT) && \
+	(CONFIG_BT_CTLR_LLL_PRIO == CONFIG_BT_CTLR_ULL_LOW_PRIO)
 	/* Enable Ticker Job, we are in a radio event which disabled it if
 	 * worker0 and job0 priority where same.
 	 */
 	mayfly_enable(TICKER_USER_ID_ULL_HIGH, TICKER_USER_ID_ULL_LOW, 1);
```

Comment:
It is not clear to me where the matching disabling happens. Could you maybe explain?

Reply:
As far as I know, disabling is done at the end of prepare, via lll_prepare_done - with the same conditions as now added to the code here.

Reply:
I does not feel quite right to have the disabling and the re-enabling on different sides of the controller split. Maybe this code should be moved into LLL?

Reply:
@wopu-ot You mean you want an LLL interface/hook that is ULL executable (say, lll_conn_ull_flush/cleanup interface) ?

Reply:
Yes, I think that would make sense, in order to have the code for disabling and enabling next to each other.